### PR TITLE
doc(observable): create observable doc file

### PR DESCRIPTION
**Pull request by fabioluz**
Hi! I created a documentation file about the `@observable`, as part of https://github.com/aurelia/framework/issues/798

I hope I haven't written anything wrong!

### CODE REVIEW

File: doc/article/en-US/binding-observable-properties.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+}
+---
+
+## [Introduction](aurelia-doc://section/1/version/1.0.0)
+
+Have you ever needed to perform an action when a property is changed? Well, that's when we use property observation. 
+
+To observe a property, you just need to decorate it with the `@observable` decorator and define a method to be the change handler. This method can receive 2 parameters, the new value and the old value. You can put any business logic inside this method.
+
+By convention, the change handler is a method whose its name is composed by the _property_name_ + 'Changed'. For example, if you decorate the property `color` with `@observable`, you have to define a method named `colorChanged()` to be the change handler. That's is how it should look:
```
Comment (StrahilKazlachev) [inline doc/article/en-US/binding-observable-properties.md]: whose *its* name => whose name

File: doc/article/en-US/binding-observable-properties.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+      colorChanged(newValue, oldValue) {
+        // this will fire whenever the 'color' property changes
+      }
+    }
+  </source-code>
+</code-listing>
+
+Note that you do not have to check if `newValue` and `oldValue` are different. The change handler will not be called if you assign a value that the property already has.
+
+If you do not want to use the conventions, you can define another name to change handler by setting the `changeHandler` property of the `@observable` decorator:
```
Comment (StrahilKazlachev) [inline doc/article/en-US/binding-observable-properties.md]: another name *to* change handler => another name *for the* change handler / another name *to* change *the* handler

File: doc/article/en-US/binding-observable-properties.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+      color = 'blue';
+
+      myChangeHandler(newValue, oldValue) {
+        // this will fire whenever the 'color' property changes
+      }
+    }
+  </source-code>
+</code-listing>
+
+You can also put the `@observable` on classes. In this case, you have to pass the property name as an argument.
```
Comment (StrahilKazlachev) [inline doc/article/en-US/binding-observable-properties.md]: From here on it does seem a bit like repetition.

**Comment by StrahilKazlachev** [comment]
You could also mention that `@observable` does track changes to the value of a property, not changes *in* the value itself - like changes in an array.
Given you are showing how to add the decorator on the class, you could show how to add it for multiple properties in this case.

**Comment by fabioluz** [comment]
I've made some changes. See if it's better now.

**Comment by bigopon** [comment]
Just my $0.02 from [How Aurelia was seen by a beginner (comment)](/aurelia/framework/issues/797#issuecomment-313903969), I think it's really good if we can get rid all of the words 'just' in Aurelia docs.

**Comment by fabioluz** [comment]
When I did this I tried to follow the same writing style present in the docs.... But you're right! Everything should be written differently. Let's see what the team will say about the new documentation, then I will rewrite this.

**Comment by EisenbergEffect** [comment]
I'll take this and do an edit pass. It probably won't go live until we launch our new site though. We're getting close to that :)

**Pull request merged**